Core/Object Arts/Dolphin/MVP/ViewTest.cls

```diff
@@ -10,6 +10,37 @@ ViewTest comment: ''!
 !ViewTest categoriesForClass!Unclassified! !
 !ViewTest methodsFor!
 
+allResourceIdentifiers
+
+	| badList list |
+	badList := self badResourceIdentifiers.
+	list := ResourceIdentifier allResourceIdentifiers.
+	list := list reject: [:each | badList includes: each printString].
+	^list!
+
+badResourceIdentifiers
+
+	^#(
+		'DatePresenter.Month view'	"#109"
+		'TextPresenter.Masked edit'	"#110"
+	)!
+
+testAllResourceIdentifiers
```

Comment:
This test will take many seconds to run, and I don't think it is going to detect many regression that matter.d It's worth keeping, but I don't think it should be run as part of the regression test suite.